kvdb-rocksdb/src/lib.rs

```diff
@@ -690,43 +702,57 @@ mod tests {
 	#[test]
 	fn get_fails_with_non_existing_column() -> io::Result<()> {
 		let db = create(1)?;
-		st::test_get_fails_with_non_existing_column(&db)
+		st::test_get_fails_with_non_existing_column(&db)?;
+		Ok(())
```

Comment:
what's the benefit of these changes?
I mean from
```rust
fun()
```
to 
```rust
fun()?;
Ok(())
```
if `fun` returns `io::Result<()>` in both cases?